packages/prismarine/src/command/vanilla/GameruleCommand.ts

```diff
@@ -26,7 +26,7 @@ export default class GameruleCommand extends Command {
                         if (value === 'true') value = true;
                         if (value === 'false') value = false;
 
-                        source.getWorld().getGameruleManager().setGamerule(name, value);
+                        source.getWorld().getGameruleManager().setGamerule(name, value, true);
```

Comment:
Why is a third boolean parameter added here? The definition for `setGamerule` is `public setGamerule(name: string, value: boolean | number): void`, a third parameter is never used.

Even with the changes its supposed to be an array with either a `number` or a `boolean` right?

Reply:
It is supposed to be Boolean it is a is editable argument added in 1.17

Reply:
This implementation is redundant as we leave these as editable arguments anyway. As from an API standpoint, a third argument "editable" is fairly confusing as it's behavior decided via the client rather than JSP. I agree with @filiphsandstrom here, and we should keep this a two param function that accepts a gamemode, and a value for that gamemode, whilst keeping everything editable.